From: Andrew Morgan <andrew@amorgan.xyz>
Date: Tue, 3 Jan 2023 14:29:53 +0000
Subject: Replace usage of inspect.getargspec with inspect.getfullargspec

It's debatable whether this check is even still needed, but for now
let's do the simple thing and update it to be compatible with modern
Python versions.
---
 bin/dh_virtualenv | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/dh_virtualenv b/bin/dh_virtualenv
index 8bafbcf..0a422ad 100755
--- a/bin/dh_virtualenv
+++ b/bin/dh_virtualenv
@@ -57,7 +57,7 @@ def main():
     # passed the packages keyword argument. Newer (like Ubuntu
     # Precise) expect the whole options to be passed.
 
-    arguments = inspect.getargspec(DebHelper.__init__).args
+    arguments = inspect.getfullargspec(DebHelper.__init__).args
     if 'packages' in arguments:
         dh = DebHelper(packages=options.package or None)
     else:
